lib/bundle/brew_services.rb

```diff
@@ -1,5 +1,16 @@
 module Bundle
   class BrewServices
+    attr_accessor :started_services
+
+    def initialize()
+      @started_services = []
+      started = `brew services list`.lines.grep(/started/)
+      for f in started
+        parts=f.split(/\s+/)
+        @started_services << Hash[:name => parts[0], :user => parts[2], :plist => parts[3]]
+      end
+    end
```

Comment:
Tue Aug  9 08:41:52 BST 2016

@MikeMcQuaid Thank you for all the feedback.  Reason for putting into ```initialize``` was to reduce having ```brew services list``` run for each formula (if moved to ```started?```).  This slows ```brew bundle dump``` to a crawl.